be/src/exec/pipeline/pipeline_driver.cpp

```diff
@@ -513,6 +516,30 @@ void PipelineDriver::_adjust_memory_usage(RuntimeState* state, MemTracker* track
     }
 }
 
+const double release_buffer_mem_threshold = 0.8;
+
+void PipelineDriver::_try_to_release_buffer(RuntimeState* state, OperatorPtr& op) {
+    if (state->enable_spill() && op->releaseable()) {
+        auto& mem_resource_mgr = op->mem_resource_manager();
+        if (mem_resource_mgr.is_releasing()) {
+            return;
+        }
+        auto query_mem_tracker = _query_ctx->mem_tracker();
+        auto query_mem_limit = query_mem_tracker->limit();
+        auto query_consumption = query_mem_tracker->consumption();
+        auto spill_mem_threshold = query_mem_limit * state->spill_mem_limit_threshold();
+        if (query_consumption >= spill_mem_threshold * release_buffer_mem_threshold) {
```

Comment:
it is confusing here,  two mem_threshold  mutiply together?  one of it must be a percentage or ratio? if two are mem threshold,  we should use max, min, avg to tie together.

Reply:
prefer rename `release_buffer_mem_threshold`

Reply:
good suggestion, I have changed it to release_buffer_mem_ratio